server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/dvchain/DVChainExchange.java

```diff
@@ -82,8 +95,37 @@ public BigDecimal getCryptoBalance(String cryptoCurrency){
         return BigDecimal.ZERO;
     }
 
+    @Override
+    public BigDecimal getFiatBalance(String fiatCurrency){
+        return BigDecimal.valueOf(10000000);
+    }
+
     @Override
     public String getDepositAddress(String cryptoCurrency){
         return null;
     }
+
+    @Override
+    public String purchaseCoins(BigDecimal amount, String cryptoCurrency, String fiatCurrencyToUse, String description) {
```

Comment:
Where does the purchased crypto end? Do you have to have a wallet address registered with the exchange?

Reply:
DV Chain manually settles up with clients daily.